nextcord/channel.py

```diff
@@ -941,6 +947,10 @@ def _update(self, guild: Guild, data: ForumChannelPayload) -> None:
             self.default_sort_order: Optional[SortOrderType] = try_enum(SortOrderType, sort_order)
         else:
             self.default_sort_order: Optional[SortOrderType] = None
+        if layout := data.get("default_forum_layout"):
```

Comment:
The docs only mention it as ommitable because it doesn't exist on other channel types. Could this default to `not_set`?